lib/buildFactory.js

```diff
@@ -142,15 +149,19 @@ class BuildFactory extends BaseFactory {
      * @param  {String}    config.eventId        The eventId that this build belongs to
      * @param  {String}    config.jobId          The job associated with this build
      * @param  {String}    config.username       The user that created this build
+     * @param  {String}    config.scmContext     The scm context to which user belongs
      * @param  {String}    [config.sha]          The sha of the build
      * @param  {String}    [config.prRef]        The PR branch or reference
      * @return {Promise}
      */
     create(config) {
         const number = Date.now();
         const modelConfig = config;
+        const displayName = this.scm.getDisplayName(config)
+            ? `${this.scm.getDisplayName(config)}:${config.username}`
```

Comment:
don't call `getDisplayName()` twice. 
`const dispayLabel = this.scm.getDisplayName(config)`
`const displayname = dispayLabel ? `${displayLabel}:${config.username}` : config.username'